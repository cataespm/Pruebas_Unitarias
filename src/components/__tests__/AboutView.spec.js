import { describe, it, expect, test } from 'vitest'

import { mount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('AboutView', () => {
  const wrapper = mount(AboutView)
  it('', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
  test(async () => {
    expect(wrapper.text()).toContain('Hello Vitest')
  })
})
