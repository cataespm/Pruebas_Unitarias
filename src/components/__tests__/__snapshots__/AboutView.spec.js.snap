// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AboutView >  1`] = `
"<div class="about">
  <h1>Hello Vitest</h1>
</div>"
`;
